docs/concept/buckconfig.soy

```diff
@@ -1570,16 +1570,34 @@ $ buck targets --resolve-alias app#src_jar
   {param name: 'jdk_name' /}
   {param example_value: 'Java SDK 1.6' /}
   {param description}
-    IntelliJ project SDK name.
+    IntelliJ project Java SDK name.
   {/param}
 {/call}
 
 {call buckconfig.entry}
   {param section: 'intellij' /}
   {param name: 'jdk_type' /}
-  {param example_value: 'Android SDK or JavaSDK' /}
+  {param example_value: 'JavaSDK' /}
   {param description}
-    IntelliJ project SDK type.
+    IntelliJ project Java SDK type.
+  {/param}
+{/call}
+
+{call buckconfig.entry}
+  {param section: 'intellij' /}
+  {param name: 'android_sdk_type' /}
+  {param example_value: 'Android SDK' /}
+  {param description}
+    IntelliJ project Android SDK type.
+  {/param}
+{/call}
+
+{call buckconfig.entry}
+  {param section: 'intellij' /}
+  {param name: 'android_sdk_name' /}
```

Comment:
Same comment as above. I think this should be named `android_sdk_name` as it applies to all android rules